nekoyume/Assets/_Scripts/Game/Game.cs

```diff
@@ -175,9 +181,17 @@ protected override void Awake()
 #if UNITY_IOS
             _commandLineOptions = CommandLineOptions.Load(Platform.GetStreamingAssetsPath("clo.json"));
 #else
-            _commandLineOptions = CommandLineOptions.Load(
-                CommandLineOptionsJsonPath
-            );
+            if (useLocalHeadless || useLocalMarketService)
+            {
+                _commandLineOptions =
+                    CommandLineOptions.Load(Platform.GetStreamingAssetsPath("clo.local.json"));
+            }
+            else
+            {
+                _commandLineOptions = CommandLineOptions.Load(
+                    CommandLineOptionsJsonPath
+                );
+            }
```

Comment:
On final touch of embedded headless, I decided not to use `clo.local.json` anymore. Instead, Unity will use same `clo.json` as so before.
This is because there is no need to to separate `clo.json` and `clo.local.json` since `Agent.cs` is gone.
If you are OK, how about using `clo.json`? Reverting this hunk will enough.